### ISSUE TITLE
Cannot create dynamic table

### ISSUE THREAD

**Issue by shay972** [bpampuch/pdfmake#713]
I have unknown table size which i wish to export to pfd file
I am unable to find a way to create the table - my issue is new line (rows)

`  var docDefinition = {`
     `pageOrientation: 'landscape',`
     `content: [`
      `{`
      `table: {`
            ` headerRows: 1,`
             `body: [  aaa ]`
           ` }`
`      }`
     `]   `     
    `};`

I tried 

`aaa.push ('Column 1', 'Column 2', 'Column 3');`
`aaa.push (','); <---- this is not working - how to I put "," between [  ], so it will go down 1 line`
`aaa.push ('text 1', 'text 2', 'text 3');`

basically I have header line, and then
I have :
    `for (var i=0 ;  i < this.AllResults.length ;  i++){`
     `aaa.push( [{ text: this.AllResults[i].GatewayModel }],);`
  ` }`

**Comment by shay972** [comment]
solved, based on answer #2 
http://stackoverflow.com/questions/26658535/building-table-dynamically-with-pdfmake
